src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -155,8 +158,15 @@ public enum TransformerVoltageControlMode {
         INCREMENTAL_VOLTAGE_CONTROL
     }
 
+    public enum ShuntVoltageControlMode {
+        WITH_GENERATOR_VOLTAGE_CONTROL,
+        INCREMENTAL_VOLTAGE_CONTROL
+    }
+
     public static final TransformerVoltageControlMode TRANSFORMER_VOLTAGE_CONTROL_MODE_DEFAULT_VALUE = TransformerVoltageControlMode.WITH_GENERATOR_VOLTAGE_CONTROL;
```

Comment:
Both default values of voltage control for transformers and for shunts can be changed in the future, if the incremental outerloops are mature enough.